modules/installation-proxy-remove.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/insall_config/enable-cluster-wide-proxy.adoc
+
+[id="installation-proxy-remove_{context}"]
+= Removing proxy from the cluster
+
+The `cluster` proxy object cannot be deleted. To remove proxy from a cluster,
+remove all `spec` fields from the proxy object.
+
+.Prerequisites
+
+* Cluster administrator permissions
+* {product-title} `oc` CLI tool installed
+
+.Procedure
+
+. Use the `oc edit` command to modify the proxy:
++
+----
+$ oc edit proxy/cluster
+----
+
+. Remove all `spec` fields from the proxy object. For example:
```

Comment:
So I think the final object is simply:

```
apiVersion: config.openshift.io/v1
kind: Proxy
metadata:
  name: cluster
spec:
  trustedCA:
    name: ""
```

Reply:
Example should be:
```
$ oc get proxy/cluster -o yaml
apiVersion: config.openshift.io/v1
kind: Proxy
metadata:
  name: cluster
spec: {}
status: {}
```

@jboxman I have an open question with the installer team regarding why this field is being set when spec should be `spec: {}`. It's harmless that the field is set to an empty string, but may cause confusion and docs inaccuracies.

Reply:
That makes a lot more sense! Thanks!